prow/cmd/branchprotector/protect.go

```diff
@@ -191,21 +184,17 @@ func (p *Protector) Protect() {
 		}
```

Comment:
I still feel like you should have done the config validation at load-time. This run-time checking for valid config input just makes this longer and harder to follow and we should be able to validate it well before it is used.

Reply:
Would you be happier if I make this a panic and/or add a comment noting that this code should never run?

Given that we're doing strings.Split() there's technically no guarantee that we'll have 2 parts, and this message will be easier to debug than an index out of bounds error on the slice on L193.

Reply:
I feel like this is just another case of where we need either:

 - users to have worse (???) UX and provide us with appropriately separated data instead of flat strings
 - internal and external config types, where the internal one here would have a struct and the type system would give us confidence in it's correctness

You're right, slice OOB would be annoying. Until we do one of the above we can keep this as-is.